src/services/chat/service.ts

```diff
@@ -32,12 +33,11 @@ export class ChatMessageService {
     // check permission
     await this.itemService.get(actor, repositories, itemId);
 
-    const messages = await chatMessageRepository.getByItem(itemId);
-    return messages;
+    return await chatMessageRepository.getByItem(itemId);
   }
 
   async postOne(
-    actor: Account,
+    actor: Guest | Member,
```

Comment:
Question: so we can never use `Account`?

Reply:
We should never be able to use an account that is neither a guest or a member.
I think it would be betther to pass the `account.id` as parameter here.